drivers/usb/device/Kconfig

```diff
@@ -158,7 +158,7 @@ config USB_DC_NXP_EHCI
 
 config USB_DC_NXP_LPCIP3511
 	bool "LPC USB Device Controller"
-	select USB_DC_HAS_HS_SUPPORT
+	select USB_DC_HAS_HS_SUPPORT if "$(dt_nodelabel_enabled,usbhs)"
```

Comment:
is there a way to do this without the need for a specifically labelled node in DT?

Reply:
I am not aware of a way.